components/search_results/search_results.jsx

```diff
@@ -47,7 +47,6 @@ export default class SearchResults extends React.PureComponent {
     static propTypes = {
         results: PropTypes.array,
         matches: PropTypes.object,
-        currentUser: PropTypes.object,
```

Comment:
We can stop passing `currentUser` from the index and remove it from the `defaultProps`